src/backend/dx11/src/lib.rs

```diff
@@ -1769,14 +1037,27 @@ unsafe impl Sync for Image { }
 #[derivative(Debug)]
 pub struct ImageView {
     #[derivative(Debug="ignore")]
-    rtv: ComPtr<d3d11::ID3D11RenderTargetView>
+    rtv_handle: Option<ComPtr<d3d11::ID3D11RenderTargetView>>,
+    #[derivative(Debug="ignore")]
```

Comment:
I don't think `derive(Debug)` makes any sense at all here given that all the fields are ignored :)

Reply:
Required by the backend trait:
```rust
type ImageView: fmt::Debug + Any + Send + Sync;
```

Supposedly you can do
```rust
#[derivative(Debug="ignore")]
struct ImageView {
    ...
}
```
on top of the struct, but then you start getting fun errors like this:

```rust
error[E0308]: mismatched types
    --> C:\Dev\gfx\src\backend\dx11\src\lib.rs:1036:10
     |
1036 | #[derive(Derivative, Clone)]
     |          ^^^^^^^^^^ expected enum `std::result::Result`, found ()
     |
     = note: expected type `std::result::Result<(), std::fmt::Error>`
                found type `()`
```